packages/@aws-cdk/aws-glue-alpha/test/job.test.ts

```diff
@@ -629,6 +630,7 @@ describe('Job', () => {
           executable: glue.JobExecutable.pythonRay({
             glueVersion: glue.GlueVersion.V4_0,
             pythonVersion: glue.PythonVersion.THREE_NINE,
+            runtime: glue.Runtime.RAY_TWO_FOUR,
```

Comment:
also need a test for when you specify a Ray job without runtime.

Reply:
Added one more test case.